conf/jetty/jetty.xml.production

```diff
@@ -614,7 +614,72 @@
 		    <Set name="replacement">%%zimbraMailURL%%</Set>
 		</New>
 	    </Arg>
-        </Call>       
+        </Call>      
+
+        <Call name="addRule">
+          <Arg>
+            <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
+              <Set name="pattern">/bundle.*</Set>
+              <Set name="name">Cache-Control</Set>
+              <Set name="value">Max-Age=86400,public</Set>
+            </New>
+          </Arg>
+        </Call>
+
+        <Call name="addRule">
+          <Arg>
+            <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
+              <Set name="pattern">.*(.chunk.*)</Set>
```

Comment:
same as https://github.com/Zimbra/zm-jetty-conf/pull/9/files#r334025445
let's restrict this for zimbrax only

Reply:
for all below rules